adoc/admin-storage-vsphere.adoc

```diff
@@ -0,0 +1,252 @@
+= vSphere Storage
+The vSphere cloud provider can be enabled with {productname} to allow Kubernetes pods to use VMWare vSphere Virtual Machine Disk (VMDK) volumes as persistent storage.
+
+This chapter provides the two types of persistent volume usage and description.
+
+Please refer to <<cluster.bootstrap.vcp>> on how to setup vSphere cloud provider enabled cluster.
```

Comment:
You can use xrefs within the same guide but not to point at other guides. This will be possible in the future but currently it doesn't work.

Please change this xref to a link to the bootstrap chapter you created. Our conversion backend can not link between separate documents and here you have an xref in the admin guide pointing at a section of the deployment guide. If you look at the Travis log this breaks the build.

You can use `{docurl}single-html/caasp-deployment/#bootstrap` (you must adjust the last bit. The ID for the anchor is made from the anchor you created so it probably will be `#_cluster_bootstrap_vcp`. Need to double check with asciidoctor conversion or I will check this before merging.